server/game/cards/13.3-PoS/EddardStark.js

```diff
@@ -0,0 +1,26 @@
+const DrawCard = require('../../drawcard.js');
+
+class EddardStark extends DrawCard {
+	setupCardAbilities(ability) {
+			this.action({
+				condition: () => this.allCharactersHaveStarkAffiliation()
```

Comment:
In order for this to work, you will need to define the `allCharactersHaveStarkAffiliation` function inside this class (but outside the `setupCardAbilities` function). You should be able to copy/paste this from similar cards.

Reply:
Thanks for the feedback, just joined discord.

Reply:
Oops, you also need a comma after this line before the title.